Resources/Private/JavaScript/Components/Frame/index.js

```diff
@@ -0,0 +1,109 @@
+// 98997e98ea6d5ececc49db0de528151e40e272b7 from https://github.com/ryanseddon/react-frame-component/commit/98997e98ea6d5ececc49db0de528151e40e272b7
```

Comment:
Hey,

I heavily patched it; we'll eventually can submit PRs upstream - but for now this was the best way to evaluate whether the idea worked :)

All the best,
Sebastian